app/interface/deps/usecases/course_deps.py

```
@@ -0,0 +1,33 @@
+) -> CourseListUseCase:
+    return CourseListUseCase(
+        year=request.year, semester=request.semester, scraper=scraper
+    )
+
+
+def get_all_courses_usecase(
+    request: AllCoursesRequest,
+    scraper: LmsScraper = Depends(get_lms_scraper, use_cache=False),
+) -> AllCoursesUseCase:
+    return AllCoursesUseCase(
+        year=request.year, semester=request.semester, scraper=scraper
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)